src/main/java/com/flair/bi/service/UserService.java

```diff
@@ -131,8 +131,8 @@ public User createUser(ManagedUserVM managedUserVM) {
         user.setPassword(encryptedPassword);
         user.setResetKey(RandomUtil.generateResetKey());
         user.setResetDate(ZonedDateTime.now());
-        user.setActivated(false);
-        user.setUserType(managedUserVM.getUserType());
+        user.setActivated(true);
```

Comment:
Arent we suppose to take this from GUI and se t it?

Reply:
Yah thats correct but by default in UI its checked and disabled so user admin cannot uncheck it.